### ISSUE TITLE
The window brightness is overexposed

### ISSUE THREAD

**Issue by ococoaocoaco** [QL-Win/QuickLook#1334]
My first use problem report
I'm not sure of the correct process and am using machine translation, so please forgive me. T-T

I haven't made any changes to the program,
I tried "msi" and "zip."
**The window is overly bright and translucent at all times**
This error appears in the preview window of any file.

**Image Links:**
https://ibb.co/CHm6B5m
https://ibb.co/NnV2xcv
https://ibb.co/2q56cND
_(Sorry, I didn't know you could send pictures directly back then.)_

OS Version: Windows10 LTSC 2019(1809) x64 [VMWARE 16 virtual machine]
QuickLook Version: The latest (3.7.3)

I don't know if it's a VM issue, but it's also just a normal VM for installing some bloated working software

**Comment by ococoaocoaco** [comment]
I proceeded to use it on other Home Edition Windows 10 VMs
They all had the problem as well
Maybe the problem comes from QuickLook not being compatible with the VMware graphics driver
uh...?

**Comment by ococoaocoaco** [comment]
I searched Github's list of issues reports for VMWARE
Found another report of a 2021 shutdown
The issues doesn't seem to be resolvable :(

If this problem is still unable to be resolved at present, then I will close this report 
I'm sorry to bother you all.

**Comment by ococoaocoaco** [comment]
LTSC system lacks its own player
I can't find a better program than QuickLook   (；′⌒`)

QuickLook is not setting,
I don't know how to change the configuration file either.
Just don't let VMware render the blur... At least try
Maybe the window will read properly.

I'm so hoping someone can help me
So I shamelessly opened the report again, pardon me ヽ(*。>Д<)o゜

**Comment by xupefei** [comment]
Yeah the main issue is VMware GPU driver. Let me try to give a fix.

**Comment by xupefei** [comment]
Oh I found that the code is already there: https://github.com/QL-Win/QuickLook/wiki/Advanced-configurations#enabledisable-window-transparency-since-v3611-23

Can you try this one?

**Comment by emako** [comment]
Outofdate
